### PR TITLE
issue #1744 Added emails whitelist that can send large attachments

### PR DESCRIPTION
**Pull request by michael-volynets**
Issue #1744 

Time Spent: 1h 30m

Went smoothly

### CODE REVIEW

**Comment by michael-volynets** [comment]
Fixing the test

**Comment by michael-volynets** [comment]
@tomholub Can we change `flowcrypt.test.trial@gmail.com` to `flowcrypt.test.trial@flowcrypt.com`?

**Comment by michael-volynets** [comment]
Even would be better to add `flowcrypt.test.trial@flowcrypt.com` as an alias there

**Comment by tomholub** [comment]
I think this involves a backend change (I have an exception there to be able to wipe this account's settings for ci). 

I could try to give it an email alias that is @somecompany

**Comment by michael-volynets** [comment]
Yes, it doesn't matter what custom domain we will use

**Comment by tomholub** [comment]
This may be a bit tricky, I'll see what I can do

**Comment by tomholub** [comment]
You could use `test.ci.compose@org.flowcrypt.com` for the trial test. Then use any `@gmail` account to verify that it lets user send bigger files. It's not perfect but it should allow us to test both scenarios.

**Comment by tomholub** [comment]
That means we can stop using flowcrypt.test.trial@flowcrypt.com completely

**Comment by tomholub** [comment]
Plus you can test the @ gmail account using a mock.

For the org.flowcrypt account, you'll have to use live Gmail.

**Comment by tomholub** [comment]
You can replace:
```js
await FlowCryptApi.hookCiAcctDelete(Config.secrets.ci_dev_account);
```
with:
```js
await FlowCryptApi.hookCiAcctDelete('test.ci.compose@org.flowcrypt.com');
```
So that the trial account gets removed before you test with that email.

**Comment by michael-volynets** [comment]
> You can replace:
> 
> ```js
> await FlowCryptApi.hookCiAcctDelete(Config.secrets.ci_dev_account);
> ```
> 
> with:
> 
> ```js
> await FlowCryptApi.hookCiAcctDelete('test.ci.compose@org.flowcrypt.com');
> ```
> 
> So that the trial account gets removed before you test with that email.

Thanks!
I was already thinking about what I can do with that

**Comment by tomholub** [comment]
The servers are still updating. Some are updated, some not yet (will be done in a few minutes). So if you get an error from backend that it's not a dev account, just try a few minutes later.

File: test/source/tests/tests/account.ts [hunk 1]
```diff
@@ -59,6 +59,15 @@ export const defineConsumerAcctTests = (testVariant: TestVariant, testWithNewBro
 
     ava.test.todo('settings > subscribe > expire > compose > footer > subscribe');
 
-  }
+  } else {
+    ava.test('compose > large file > gmail account (regular user)', testWithSemaphoredGlobalBrowser('compose', async (t, browser) => {
```
Comment (tomholub) [inline test/source/tests/tests/account.ts]: Looks good, just for clarity let's call it `compose > large file > public domain account (should not prompt to upgrade)`

Comment (michael-volynets) [reply test/source/tests/tests/account.ts]: Okay, I'm renaming it

**Comment by tomholub** [comment]
Our ci secrets missing pwd again, I think

**Comment by tomholub** [comment]
I updated it, hopefully now it goes through

**Comment by tomholub** [comment]
image

**Comment by tomholub** [comment]
I can just start sending all test failures to your email as well. You will just be getting a bunch of emails that are not related to your work

**Comment by michael-volynets** [comment]
no problem, you can send them to me

**Comment by michael-volynets** [comment]
Sorry, that was my mistake, I was using `compose` account instead of `compatibility`

**Comment by michael-volynets** [comment]
I don't know how I could have missed that.

but the first test (`[standalone] compose > large file > subscribe > trial > attach again`) works on my side.
Is there something wrong with semaphoreci?

**Comment by tomholub** [comment]
Yes, new secrets missing in legacy semaphore setup

**Comment by tomholub** [comment]
Hopefully now it will pass

**Pull request merged**